README.md

```diff
@@ -132,6 +132,20 @@ Future<void> testExecutable(FutureOr<void> Function() testMain) async {
 
 As an alternative you can use [Alchemist](https://pub.dev/packages/alchemist).
 
+Also, you can configure `AdaptiveTestConfiguration` singleton to skip tests instead of throwing if they are run on an unintended platform.
+```dart
+Future<void> testExecutable(FutureOr<void> Function() testMain) async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  AdaptiveTestConfiguration.instance
+    ..setSkippedTestPlatform(TargetPlatform.macOS)
+    ..setFailTestOnWrontPlatform(false) <-- Adding this will skip the `testAdaptiveWidgets` tests if you are not running the tests on a macOS platform.
```

Comment:
```suggestion
    ..setFailTestOnWrongPlatform(false) <-- Adding this will skip the `testAdaptiveWidgets` tests if you are not running the tests on a macOS platform.
```

Reply:
Thanks for noticing.
Fixed ✅